roles/roles.py

```diff
@@ -0,0 +1,51 @@
+"""
+Definition of roles and their permissions
+"""
+
+from rolepermissions.roles import AbstractUserRole
+
+from roles.models import Role
+
+
+class Permissions:
+    """
+    All MicroMasters custom permissions
+    """
+    CAN_ADVANCE_SEARCH = 'can_advance_search'
+    CAN_EDIT_CMS = 'can_edit_cms'
+    CAN_MESSAGE_LEARNERS = 'can_message_learners'
+    CAN_CREATE_FORUMS = 'can_create_forums'
```

Comment:
Because `roles/permissions.py` is something else: it is the definition of the per-object permissions function checkers